modules/client-configuration/nav-client-configuration-guide.adoc

```diff
@@ -37,9 +37,15 @@ endif::[]
 **** xref:contact-methods-pushssh.adoc[Push via SSH]
 // SLE Client Registration
 ** xref:registration-overview.adoc[Client Registration]
-*** xref:registration-webui.adoc[Register with the Web UI]
-*** xref:registration-bootstrap.adoc[Register with Bootstrap]
-*** xref:registration-cli.adoc[Register on the Command Line]
+*** xref:registration-methods.adoc[Registration Methods]
+**** xref:registration-webui.adoc[Register with the Web UI]
+**** xref:registration-bootstrap.adoc[Register with Bootstrap]
+**** xref:registration-cli.adoc[Register on the Command Line]
+// Registering Clients on a Proxy
+*** xref:client-proxy.adoc[Client Registration on a Proxy]
```

Comment:
registration *on* / register *to*  --is this intentionally?

Reply:
Good point, it should be `to`.